src/cdk/drag-drop/drag-ref.ts

```diff
@@ -1237,7 +1237,7 @@ export class DragRef<T = any> {
     // Cache the previous transform amount only after the first drag sequence, because
     // we don't want our own transforms to stack on top of each other.
     if (this._initialTransform == null) {
-      this._initialTransform = this._rootElement.style.transform || '';
+      this._initialTransform =  this._rootElement.style.transform == 'none' ? '' this._rootElement.style.transform;
```

Comment:
I think that this won't handle the case where `style.transform` is undefined.

Reply:
ok I'll fix that and create a new PR with test case for this. I'll close this one. thanx @crisbeto